### PR TITLE
Rugby scorers

### PR DESCRIPTION
**Pull request by jennysivapalan**
Paired with @OliverJAsh to create score events per team for rugby :rugby_football: :trophy: 
- Parsing a score stats match feed (one per match). This requires an update in the url to Platform project as it is a different endpoint to the other feeds.
- Create two agents - one to update live games every minute and one to update results to matches every 30 minutes

screen shot 2015-08-26 at 16 51 56
- [x] Merge and deploy https://github.com/guardian/platform/pull/812
- [x] Only show score events on desktop

### CODE REVIEW

**Comment by OliverJAsh** [comment]
Depends on https://github.com/guardian/platform/pull/812

File: sport/app/rugby/views/fragments/matchSummary.scala.html [hunk 1]
```diff
@@ -41,4 +69,13 @@ <h2 class="team__heading">
             </div>
         </div>
     </div>
+    <div>
+        @if(homeTeamScorers.size > 0) {
```
Comment (OliverJAsh) [inline sport/app/rugby/views/fragments/matchSummary.scala.html]: `|| awayTeamScorers.size > 0`?

Comment (OliverJAsh) [inline sport/app/rugby/views/fragments/matchSummary.scala.html]: `(homeTeamScorers ++ awayTeamScorers).size > 0`?

Comment (jennysivapalan) [inline sport/app/rugby/views/fragments/matchSummary.scala.html]: :+1:

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**